tests/end_to_end/CairoPrecompiles/test_call_cairo_precompile.py

```diff
@@ -101,9 +101,7 @@ async def test_should_fail_when_called_with_delegatecall(
         async def test_should_fail_when_called_with_callcode(
             self, cairo_counter_caller
         ):
-            with cairo_error(
-                "EVM tx reverted, reverting SN tx because of previous calls to cairo precompiles"
-            ):
+            with evm_error("CairoLib: call_contract failed with"):
```

Comment:
This cannot stay like this. I do not understand the entire error 
```
b'\x08\xc3y\xa0\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00%CairoLib: call_contract failed with: \x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

@enitrat if you have inputs

Reply:
It is the error from the contract encoded